docs/UserGuide.md

```diff
@@ -69,14 +66,14 @@ Please note the following symbols used in the User Guide which may serve as poin
 
 ### 2.2. StoreMando's layout
 
-1. Main Panel
+1. Main Panel <br>
```

Comment:
If you are putting a line break then i suggest you give it #### for the sub headers :)

Reply:
All right noted!